test/AbstractPluginManagerTest.php

```diff
@@ -365,7 +365,7 @@ public function testValidateWillFallBackToValidatePluginWhenDefinedAndEmitDeprec
         $pluginManager->assertion = $assertion;
 
         $errorHandlerCalled = false;
-        set_error_handler(function ($errno, $errmsg) use (&$errorHandlerCalled) {
+        set_error_handler(function (int $errno, string $errmsg) use (&$errorHandlerCalled) {
```

Comment:
```suggestion
        set_error_handler(function (int $errno, string $errmsg) use (&$errorHandlerCalled): void {
```

Add the return type.